File: src/targets/javascript/jquery/fixtures/multipart-form-data.js
const form = new FormData();
form.append('foo', 'bar');

const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'POST',
  headers: {},
  processData: false,
  contentType: false,
  mimeType: 'multipart/form-data',
  data: form
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/clojure/clj_http/fixtures/multipart-form-data.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:multipart [{:name "foo"
                                                          :content "bar"}]})
=======
(client/post "http://mockbin.com/har" {:multipart [{:name "foo"
                                                    :content "bar"}]})
>>>>>>> upstream/master
